### PR TITLE
Add NightFury546's file

### PR DESCRIPTION
**Pull request by ManojKumarYekollu**
I added an introduction slide to the project.

### CODE REVIEW

File: _posts/0000-01-02-NightFury546.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
```
Comment (NightFury546) [inline _posts/0000-01-02-NightFury546.md]: Thanks @NightFury546 for this change

File: _posts/0000-01-02-NightFury546.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+---
+Your text
+Use the left arrow to go back!
```
Comment (NightFury546) [inline _posts/0000-01-02-NightFury546.md]: looks like this feature is not working. @NightFury546  Please check

**Review by ManojKumarYekollu — COMMENTED** [review]
Thanks working now

**Comment by ManojKumarYekollu** [comment]
closed

**Pull request merged**